tests/common/config_reload.py

```diff
@@ -93,3 +95,7 @@ def config_reload(duthost, config_source='config_db', wait=120, start_bgp=True,
                     "PFC_WD is missing in CONFIG-DB")
     else:
         time.sleep(wait)
+
+    if check_intf_up_ports:
```

Comment:
Is there a case where it makes sense to use `check_intf_up_ports`, but `safe_reload` is not used? If not, it might be better to just have it within the `if safe_reload` block.